plugins/kapt3/kapt3-base/src/org/jetbrains/kotlin/kapt3/base/util/java9Utils.kt

```diff
@@ -24,6 +24,16 @@ import com.sun.tools.javac.util.List as JavacList
 import org.jetbrains.kotlin.kapt3.base.plus
 
 fun isJava9OrLater(): Boolean = !System.getProperty("java.version").startsWith("1.")
+fun isJava11OrLater(): Boolean {
```

Comment:
Alternatively,
```kotlin
fun isJava11OrLater(): Boolean = try {
  Runtime.Version.parse(System.getProperty("java.version")).major() >= 11
} catch (_: NoClassDefFoundError) {
  false
}
```

Reply:
Same could be done for the above method

Reply:
The issue is that KAPT compiles against JDK8, and `Runtime.Version` is added in JDK9. Once it migrates to JDK9, using this is definitely better.